sql/hive-thriftserver/pom.xml

```diff
@@ -106,12 +111,6 @@
               </sources>
             </configuration>
           </execution>
-        </executions>
```

Comment:
This isn't technically related, but is a simple fix for a build warning, and was editing the file anyway